src/Haskell/Ide/Engine/Plugin/HfaAlign.hs

```diff
@@ -28,10 +28,8 @@ import qualified Safe
 hfaAlignDescriptor :: PluginId -> PluginDescriptor
 hfaAlignDescriptor plId = PluginDescriptor
   { pluginId = plId
-  , pluginName = "Align Equals"
-  , pluginDesc = "An example of writing an HIE plugin\nbased on http://www.haskellforall.com/2018/10/detailed-walkthrough-for-beginner.html"
```

Comment:
Can you convert the pluginDesc into a comment in the file? It gives context, for how to take a feature description and implement it in hie. Which is the point of the example.

Reply:
That’s a good point. I’ll insert it as a haddock doc for the module